src/qml/bitcoin.cpp

```diff
@@ -185,6 +186,9 @@ int QmlGuiMain(int argc, char* argv[])
 
     engine.rootContext()->setContextProperty("nodeModel", &node_model);
 
+    OptionsQmlModel options_model{*node};
+    engine.rootContext()->setContextProperty("options", &options_model);
```

Comment:
Might want a more specfic variable name here since its registered to the root to prevent possible conflicts with other property names.  "optionsModel" maybe.  

Also, I can't get my head around how this is working now. When hebasto did the previous PR setting a rootContext like this wasn't accessible in any of the sub components but this one is working. All I can think of is that it has something to do with how the StackView in main.qml is implemented now.

Reply:
In **debug.log**, I can observe a few **TypeErrors**.

```
% cat debug.log | grep GUI

2022-12-20T13:45:16Z GUI: qrc:/qml/components/StorageSettings.qml:24: TypeError: Cannot read property 'pruneSizeGB' of null
2022-12-20T13:45:16Z GUI: qrc:/qml/components/StorageSettings.qml:16: TypeError: Cannot read property 'prune' of null
```

The reason for this **TypeError** is that the `OptionsQmlModel` is initialized after the engine.
And since objects are destroyed in reverse order of how they are initialized, the `OptionsQmlModel` is getting destroyed before the QML engine.

The same **TypeErrors** existed in #184, as pointed out by @johnny9 [here](https://github.com/bitcoin-core/gui-qml/pull/184#issuecomment-1293891380).

So you have to initialize the `OptionsQmlModel` before the engine to solve this.

Reply:
will address both of these soon